src/dotnet/OpenAPI.Validator.Tests/OpenAPIModelerValidationTests.cs

```diff
@@ -907,6 +915,13 @@ public void ValidXMSPathWithOAata()
             }
             Assert.Empty(message);
         }
+
+        [Fact]
+        public void ValidRequiredDesciminatorPropertiesInPatchRequestValidation()
```

Comment:
do you have the existing sample to verify R2016? I don't think we need to create a new test case for this.

Reply:
this case is a litter special, so I add a new to test it
Btw I think we should modify one test case only when it has some bug

Reply:
Not necessary. But anyway, you could make the decision for this.